python/paddle/fluid/layers/nn.py

```diff
@@ -9313,7 +9313,7 @@ def elu(x, alpha=1.0, name=None):
 
 
 @templatedoc()
-def relu6(x, threshold=6.0, name=None):
+def relu6(x, threshold=6.0, name=None, use_mkldnn=False):
```

Comment:
Why do you add the attribute `use_mkldnn` here?
- If we use `FLAGS_use_mkldnn=true` to global control the model, we don't need add `use_mkldnn` attribute in each API.
- #25745  sigmoid mkldnn op does not add the attribute `use_mkldnn`

Reply:
### Why we add use_mkldnn here
This Relu6 is used in `models` repo. It is needed for mobilenetv2 here https://github.com/PaddlePaddle/models/blob/develop/dygraph/mobilenet/mobilenet_v2.py#L69.

We wouldn't have to do this if `models` repo would use the approach as here: 
https://github.com/PaddlePaddle/models/blob/develop/dygraph/resnet/train.py#L207
which calls:
https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/fluid/dygraph/nn.py#L233.
which is passed to convolution here: https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/fluid/dygraph/nn.py#L170

There are few ways how ops are added in models repo, for example this: https://github.com/PaddlePaddle/models/blob/develop/dygraph/resnet/train.py#L243
I am guessing that this is inconsistent because it isn't fully migrated yet but I am not sure.

Value of use_mkldnn has to be evaluated before this line for dygraph to use mkldnn kernel of relu6: https://github.com/PaddlePaddle/Paddle/pull/25713/files/3edaaad6bc84ed19715aaca04f9ed6bdbf06e25e#diff-f8fc1afdc1f06d4c47bc0933d53bf3c6R9359.

### Old vs New approach
This `use_mkldnn=False` is from previous approach before we decided ot add `FLAGS_use_mkldnn`.
The new approach we have is to pass `use_mkldnn=True` to ops and check it with `FLAGS_use_mkldnn`. For details please refer to this PR
https://github.com/PaddlePaddle/Paddle/pull/25773/files#diff-aeec709d2310c344dd0f235478865b7cR182. We pass use_mkldnn=True and we call `CheckUseMkldnnWithFlag()` to compare against `FLAGS_use_mkldnn`.

### How we can proceed
We could apply the same approach to this PR with checking against FLAGS_use_mkldnn. If you think this is correct, I can do that here.
Alternatively, maybe models repo could call relu6 in different way? Or maybe you have other ideas?

Reply:
I found the same situation here: https://github.com/PaddlePaddle/Paddle/pull/25773#discussion_r461595326
We should definitely decide how to deal with that so I can change both places.

Reply:
> We could apply the same approach to this PR with checking against FLAGS_use_mkldnn. If you think this is correct, I can do that here.
Alternatively, maybe models repo could call relu6 in different way? Or maybe you have other ideas?

@phlrain Please have a look, and which way do you prefer?

Reply:
I prefer to use FLAGS_use_mkldnn. It's seems an easier way